### ISSUE TITLE
Can't build on Windows 7

### ISSUE THREAD

**Issue by bdenison99** [dotnet/corefx#8404]
Could this be caused by my running a build on Windows 7?
I see the following in the output, and not long after I get lots of errors:

C:\Users\bdenison\git_workspace\net_core\Tools\packageresolve.targets(78,5): warning : Your project.json doesn't have a runtimes section. You should add '"runtimes": { "win7-x64":
{ } }' to your project.json and then re-run NuGet restore. [C:\Users\bdenison\git_workspace\net_core\src\System.AppContext\src\redist\System.AppContext.depproj]
C:\Users\bdenison\git_workspace\net_core\Tools\packageresolve.targets(78,5): warning : Your project.json doesn't have a runtimes section. You should add '"runtimes": { "win7-x64":
{ } }' to your project.json and then re-run NuGet restore. [C:\Users\bdenison\git_workspace\net_core\src\System.AppContext\src\redist\System.AppContext.depproj]

...

C:\Users\bdenison\git_workspace\net_core\Tools\packageresolve.targets(78,5): warning : Your project is not referencing the "DNXCore,Version=v5.0" framework. Add a reference to "DN
XCore,Version=v5.0" in the "frameworks" section of your project.json, and then re-run NuGet restore. [C:\Users\bdenison\git_workspace\net_core\src\System.Data.SqlClient\src\System
.Data.SqlClient.csproj]
C:\Users\bdenison\git_workspace\net_core\Tools\packageresolve.targets(78,5): warning : Your project is not referencing the "DNXCore,Version=v5.0" framework. Add a reference to "DN
XCore,Version=v5.0" in the "frameworks" section of your project.json, and then re-run NuGet restore. [C:\Users\bdenison\git_workspace\net_core\src\System.Data.SqlClient\src\System
.Data.SqlClient.csproj]
C:\Users\bdenison\git_workspace\net_core\Tools\packageresolve.targets(78,5): warning : Your project is not referencing the "DNXCore,Version=v5.0" framework. Add a reference to "DN
XCore,Version=v5.0" in the "frameworks" section of your project.json, and then re-run NuGet restore. [C:\Users\bdenison\git_workspace\net_core\src\System.Data.SqlClient\src\System
.Data.SqlClient.csproj]
C:\Users\bdenison\git_workspace\net_core\Tools\packageresolve.targets(78,5): warning : Your project is not referencing the "DNXCore,Version=v5.0" framework. Add a reference to "DN
XCore,Version=v5.0" in the "frameworks" section of your project.json, and then re-run NuGet restore. [C:\Users\bdenison\git_workspace\net_core\src\System.Data.SqlClient\src\System
.Data.SqlClient.csproj]

...

C:\Users\bdenison\git_workspace\net_core\Tools\tests.targets(190,5): error MSB4018: The "GetTargetMachineInfo" task failed unexpectedly.\r [C:\Users\bdenison\git_workspace\net_cor
e\src\Common\tests\Common.Tests.csproj]
C:\Users\bdenison\git_workspace\net_core\Tools\tests.targets(190,5): error MSB4018: System.DllNotFoundException: Unable to load DLL 'api-ms-win-core-sysinfo-l1-2-0.dll': The speci
fied module could not be found. (Exception from HRESULT: 0x8007007E)\r C:\Users\bdenison\git_workspace\net_core\src\Common\tests\Common.Tests.csproj

**Comment by ellismg** [comment]
This may be the same as https://github.com/dotnet/corefx/issues/8291.  A fix went in recently to master.  What SHA is your HEAD at?

**Comment by bdenison99** [comment]
acd7df39cff151135f8504dc5ea7bcf98c47d1e6

**Comment by bdenison99** [comment]
It's at "**acd7df39cff151135f8504dc5ea7bcf98c47d1e6**"

**Comment by ellismg** [comment]
@Priya91 Can you please take a look?

**Comment by bdenison99** [comment]
I've just updated my local repo with the last 4 checkins.  SHA is now at b9cdde3843b410ac6696513113a40743c39f2fb2 and has significantly fewer errors, but still fails to build cleanly.  It's down to 7 errors / 29 warnings.

**Comment by Priya91** [comment]
Yes, this should be fixed. Can you do this and try again..

```
git clean -xdf
.\clean /all
.\build
```

**Comment by bdenison99** [comment]
I have no untracked changes.  Attached is the build log from this morning, after pulling all changes made since yesterday.
[bart_build.txt](https://github.com/dotnet/corefx/files/257678/bart_build.txt)

**Comment by Priya91** [comment]
The build is not reporting the GetTargetMachineInfo errors from the log, but there are some test failures. Looks like you're unblocked now. Is there any other concern?

**Comment by bdenison99** [comment]
I think it looks a lot better.  I'll work on some of the code projects later and see if there is another issue.  Feel free to close this issue, since I can always open another one later if I run into additional problems.

**Comment by Priya91** [comment]
Okie, thanks!
